### PR TITLE
[Simulators] Improve backtesting support 

### PR DESCRIPTION
**Pull request by Herklos**
Depends on https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/59

### CODE REVIEW

File: octobot_trading/channels/exchange_channel.py [hunk 1]
```diff
@@ -15,52 +15,44 @@
...
-        self.logger.debug("Pausing...")
-        # Triggers itself if not already paused
-        if not self.channel.is_paused:
-            self.channel.is_paused = True
-
-    async def resume(self) -> None:
-        self.logger.debug("Resuming...")
-        # Triggers itself if not already resumed
-        if self.channel.is_paused:
-            self.channel.is_paused = False
```
Comment (GuillaumeDSM) [inline octobot_trading/channels/exchange_channel.py:55]: not required anymore ?

File: octobot_trading/producers/balance_updater.py [hunk 2]
```diff
@@ -20,73 +20,139 @@
...
 
-    async def start(self):
+    async def start(self) -> None:
+        """
+        Starts the balance updating process
+        """
         while not self.should_stop and not self.channel.is_paused:
             try:
-                await self.push((await self.channel.exchange_manager.exchange.get_balance()))
+                await self.push((await self.fetch_portfolio()))
```
Comment (GuillaumeDSM) [inline octobot_trading/producers/balance_updater.py:51]: 👍

File: octobot_trading/channels/exchange_channel.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/channels/exchange_channel.py:55]: It's only unnecessary overrides

File: tests/data/test_portfolio.py [hunk 3]
```diff
@@ -691,6 +691,7 @@ async def test_update_portfolio_with_multiple_symbols_orders(self):
         # test buy order creation
         portfolio_manager.portfolio.update_portfolio_available(market_buy, True)
         assert portfolio_manager.portfolio.get_currency_portfolio("LTC", PORTFOLIO_AVAILABLE) == 0
+        print(portfolio_manager.portfolio.portfolio)
```
Comment (GuillaumeDSM) [inline tests/data/test_portfolio.py:694]: ?

**Pull request merged**